tests/lifecycle/driver_test.go

```diff
@@ -55,7 +55,7 @@ func TestServices(t *testing.T) {
 
 func TestMain(m *testing.M) {
 	if err := crypto.InitializeGlobalCodec(noop.NewCodec()); err != nil {
-		panic(err)
+		os.Exit(-1)
```

Comment:
Why change to `os.Exit(1)` here? Seems `panic(err)` can gives more information.

Reply:
Yes. But it should follow other exits in this function. FYI: https://stackoverflow.com/questions/28472922/when-to-use-os-exit-and-panic.